src/include/usr/hwpf/hwp/mvpd_accessors/getMBvpdAttr.H

```diff
@@ -227,6 +227,13 @@ namespace getAttrData
         uint8_t              iv_systemType;
         uint8_t              iv_systemType_ext;
         uint8_t              iv_dataVersion;
+    public:
```

Comment:
Yes, without this patch I got (as the commit message for 99e40658a72dcdae88ce8bd3814c4aec50683cdd states): " error: dereferencing type-punned pointer will break strict-aliasing rules [-Werror=strict-aliasing]"

Which was due to alternately treating it as u32, u8[] and MBvpdVMKeyword. I didn't check the asm, but I would assume that the compiler is smart enough to effectively compile down the code to nothing.